src/fu-engine.c

```diff
@@ -2651,7 +2651,7 @@ fu_engine_create_metadata (FuEngine *self, XbBuilder *builder,
 		const gchar *fn = g_ptr_array_index (files, i);
 
 		/* check is cab file */
-		if (!g_str_has_suffix (fn, ".cab")) {
+		if (!g_str_has_suffix (g_ascii_strdown(fn, -1), ".cab")) {
```

Comment:
`g_ascii_strdown` returns allocated memory. Would this be suitable instead?

    if (!g_str_has_suffix (fn, ".cab") &&
        !g_str_has_suffix (fn, ".CAB")) {

Reply:
That of course wouldn't cover Cab or cAB though too.  I think if those are important to cover you'll need to do something above like

`g_autofree gchar * fn_lowercase = g_ascii_strdown(fn, -1);`

and then use `fn_lowercase` in this check.

Reply:
Thanks for comments, I've made the changes.